### DISCUSSION TITLE
default yml files missing?

### DISCUSSION THREAD

**Discussion by flaminiom** [meisnate12/Plex-Meta-Manager-Configs#598] [Q&A]
I've got some yml files that are missing from the default install. I'm wondering a strategy for dealing with this. PMM I assume is looking in /app/pmm/defaults/overlays/ inside the container. Adding them to my appdata config folder (unraid) doesn't do anything. 

I think if I put a file in the docker defaults it will just overwrite when reloaded?

File Error: Default does not exist genre.yml                                               |
File Error: Default does not exist year.yml                                                |
File Error: Default does not exist decade.yml                                              |
File Error: Default does not exist seasonal.yml                                            |
File Error: Default does not exist universe.yml

**Comment by minermartijn** [comment]
I now have the same for Actor.

`File Error: Default does not exist actor.yml`
